### ISSUE TITLE
WPT Dropdown list is not working

### ISSUE THREAD

**Issue by jayaram250** [WPO-Foundation/webpagetest#1563]
## Description
we are using webpagetest,wpt-agent in gce.

From three days onwards we are unable to use the WPT dropdown list is not working. Please find the attached screenshot below. 

We didn't change anything from our end. 

Running agents also not showing in /getLocations.php?f=html&hidden=1 or /getTesters.php?f=html&hidden=1

![WPT-Dropdown List not working](https://user-images.githubusercontent.com/35720482/135805614-b2faab3c-bf72-4c23-9997-d5a40e7bb225.png)

**Comment by saurabh4sharma21** [comment]
I am also facing same issue. Nothing was changed at my end.

**Comment by talekardj** [comment]
Yeah. Facing the same issue. No changes from our end and suddenly this started happening. Please look into it urgently.

**Comment by rposbo** [comment]
This is **possibly** related to an issue (started about 3 days ago) where the certificates for the nodejs installation deb package changed/expired which means the wptagent installation **fails**. 

This means any _new_ agents will fail to start, since nodejs fails to install. This is a known issue (I'm trying to find the reference), and without some hacks on the wptagent install script (e.g. to pass `-k` to ignore certificate issues during install) there's not much you can do.

**Comment by tkadlec** [comment]
Thanks for the additional context, @rposbo ! Given the timing, I was wondering if it was related to all the cert issues. 

@talekardj @saurabh4sharma21 @jayaram250 This seems like a _super_ likely culprit, as nothing related to this changed on our end either. I can look into how we might be able to put a workaround in place, or at least suggest a temporary solution, but first, does this seem to add up? Do you have any logs you could look into for the agents?

**Comment by chandanpasunoori** [comment]
@tkadlec I can confirm its nodesource ssl issue, below log is printed repeatedly in agent screen in agent node.

Err:7 https://deb.nodesource.com/node_12.x bionic Release
  Certificate verification failed: The certificate is NOT trusted.

**Comment by chandanpasunoori** [comment]
@tkadlec I can confirm that issued fixed by manually running `sudo apt install ca-certificates` followed by `sudo apt update`

**Comment by chandanpasunoori** [comment]
@tkadlec @rposbo  for now I have updated gce agent installation script like below, adding `apt update` and `ca-certificates` installation. created new image and updated template. agents are registering to server now. 

`WPT_UPDATE_OS` is already true, so had to add before first run and agent scripts.

if these changes are fine without any side effects, let me know I can create PR. 

```bash
# build the startup script
echo '#!/bin/sh' > ~/startup.sh
echo "PATH=$PWD/bin:$PWD/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin" >> ~/startup.sh
echo 'sudo DEBIAN_FRONTEND=noninteractive apt update -yq' >> ~/startup.sh
echo 'sudo DEBIAN_FRONTEND=noninteractive apt install ca-certificates -yq' >> ~/startup.sh
echo 'cd ~' >> ~/startup.sh
echo 'if [ -e first.run ]' >> ~/startup.sh
echo 'then' >> ~/startup.sh
echo '    screen -dmS init ~/firstrun.sh' >> ~/startup.sh
echo 'else' >> ~/startup.sh
echo '    screen -dmS agent ~/agent.sh' >> ~/startup.sh
echo 'fi' >> ~/startup.sh
echo 'sudo service watchdog restart' >> ~/startup.sh
chmod +x ~/startup.sh
```

added lines 
```bash
echo 'sudo DEBIAN_FRONTEND=noninteractive apt update -yq' >> ~/startup.sh
echo 'sudo DEBIAN_FRONTEND=noninteractive apt install ca-certificates -yq' >> ~/startup.sh
```

**Comment by TKaram** [comment]
> @tkadlec I can confirm that issue fixed by manually running `sudo apt install ca-certificates` followed by `sudo apt update`

Hello,
If I have multiple agents running, do I have to run the command on each of the agents or I must forget about theses agents and set up new one from the beginning ?
Tkank you

**Comment by TKaram** [comment]
Updating the libgnutls30 package appears to fix the certificate verification :
`sudo apt-get install libgnutls30`
`sudo apt-get update`
I just had to run these command on each of my agents (VMs on GCE) to make it work
